tools/loader/ImageClassifier.cpp

```diff
@@ -77,6 +77,12 @@ llvm::cl::opt<bool> convertInAndOutToFp16(
     llvm::cl::desc(
         "Convert the input and output tensors of the network to fp16"),
     llvm::cl::cat(imageLoaderCat));
+
+llvm::cl::opt<std::string> expectedMatchingIndices(
```

Comment:
Instead of using a string, you could just make this a `llvm::cl::list<int>` and then specify it as `llvm::cl::CommaSeparated`. Then you wouldn't need to split the string or convert to int.

Reply:
Thanks Jordan for great tip!

Reply:
Fixed in the last commit